.flake8

```diff
@@ -60,14 +60,32 @@ max-line-length = 100
 
 # Allow certain violations in certain files:
 per-file-ignores =
-  docs/conf.py: WPS221, WPS226, WPS301, WPS305, WPS323, WPS420, WPS429, WPS433
+  # The following ignores have been researched and should be considered permanent
+  # each should be preceeded with an explanation of each of the error codes
+  # These might be commented out as they overlap with the perfile ignores
+  # once the per file ignores are removed or added within the file itself
+  # they should be able to be uncommented
+
+  # S101: Allow the use of assert within the tests directory
+  # tests/*: S101
```

Comment:
Just uncomment it right now. The duplication won't make a difference. But when the per-file ignores are 
```suggestion
  tests/**.py: S101
```

Here's what I recommend usually — have all of the ignores necessary and explain why it makes sense to ignore each in a comment on top of that section: https://github.com/ansible-community/ansible-lint/blob/718e4cd/.flake8#L168.